bin/index.js

```diff
@@ -104,50 +101,25 @@ const askQuestions2 = () => {
       type: 'list',
       name: '_actionType',
       message: 'What action is the certificate asserting?',
-      choices: ["REGISTER", "UPDATE"],
+      choices: ["REGISTER", "DE_REGISTER"],
       default: "REGISTER"
     },
     {
-      name: "_releaseNumber",
+      name: "_releaseComment",
       type: "input",
-      message: "What's the release number?",
-      default: '0.0.1'
+      message: "Enter a release comment:",
+      default: 'New release'
```

Comment:
I would remove a default value as this field is optional

Reply:
Fixed on d41e940